docs/examples/example_feature_importance_shap.ipynb

```diff
@@ -367,7 +368,23 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "For guidelines on how to interpret such SHAP plots please see the [SHAP documentation](https://github.com/shap/shap).\n",
+    "In these SHAP summary plots, the color and orientation of the plotted values help understand\n",
+    "their impact on model predictions.\n",
+    "\n",
+    "Each dot in the plot represents a single instance of the given feature present in the data set.\n",
+    "The x-axis conveys the Shapley value, signifying the strength and directionality of the\n",
```

Comment:
The x-axis displays

Reply:
[6444e57](https://github.com/Quantco/metalearners/pull/21/commits/6444e579c3f3ed9708f8c3b0a151bea3a27e39ae)